js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetGraphExplorerFilters.tsx

```diff
@@ -54,7 +71,7 @@ export function useAssetGraphExplorerFilters({
     setSelectAllFilters,
   } = assetFilterState || defaultState;
```

Comment:
Basically `defaultState` didn't actually match `AssetFilterState` before so that led to bugs, not it actually matches so we can rely on type checking preventing the bugs we saw